src/services/country.js

```diff
@@ -0,0 +1,6 @@
+import { checkError, client } from './client';
```

Comment:
It is standard to name the services file with a plural. In this case, that would mean countries.js